samples/Extensions/local.settings.json

```diff
@@ -4,8 +4,9 @@
     "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
     "AzureWebJobsStorage": "",
     "CosmosConnection": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
+    "SignalRConnection": "",
```

Comment:
Did you mean to have empty string value here?

Reply:
No, a valid connection string is required to run the sample. I think it's like the `AzureWebJobsStorage` which also leaves an empty string for user to fill.